nistats/first_level_model.py

```diff
@@ -261,6 +261,14 @@ class FirstLevelModel(BaseEstimator, TransformerMixin, CacheMixin):
         This id will be used to identify a `FirstLevelModel` when passed to
         a `SecondLevelModel` object.
 
+    low_pass: None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details. [1]
```

Comment:
Sorry, i don' like such reference is dostrings, as I usually look at them in interactive mode.

Reply:
Is that true for all users? I use IDEs and appreciate if I can click on a link in the docstring for more details instead of typing it in a browser.

I am sure there are many who will do it your way and many who will like the hyperlink.

Also, does having the hyperlink interfere with the help display in interactive mode somehow? If not, then  having this link is compatible with both preferences.